app/views/portal/client_logins/link_sent.html.erb

```diff
@@ -1,9 +1,22 @@
 <% content_for :page_title, t(".title") %>
 
 <% content_for :card do %>
+  <div class="question__illustration spacing-above-0">
+    <%= image_tag("portal/confirmation-link.svg", alt: "") %>
+  </div>
+
   <h1 class="h2"><%= t(".title") %></h1>
 
-  <p><%= t(".link_sent") %></p>
+  <% if @email_address.present? || @sms_phone_number.present? %>
+    <p class="spacing-below-15"><%= t(".link_sent_at") %></p>
+  <% else %>
+    <p class="spacing-below-15"><%= t(".link_sent") %></p>
+  <% end %>
+
+  <p class="spacing-below-15 text--bold"><%= @email_address %></p>
```

Comment:
when i see the same spacing being added to a bunch of elements, to me it would make more sense to add a wrapper around this and style the p tags in css rather than with these custom classes

Reply:
I see what you mean. I'm going to leave this as-is for now, but I may do that in the future.